sql/executor.go

```diff
@@ -282,6 +282,14 @@ func runExec(slct string, db *DB) *PipeReader {
 	return rd
 }
 
+func isUnsupervisedLearning(pr *extendedSelect) bool {
```

Comment:
How about adding a unit test?

Reply:
I agree to add one, preferably in a followup pull request.